### PR TITLE
Add plugin for VC service using trusted-issuers-list

### PR DESCRIPTION
**Pull request by dwendland**

### CODE REVIEW

File: vc-trusted-issuer-data-service/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "name": "NGSI-LD Data Service for VC at trusted-issuer list",
+    "author": "dwendland",
+    "version": "0.0.1",
+    "module": "vc_til_service.VcTilService",
+    "media_types": ["FIWARE NGSI-LD"],
+    "formats": ["URL"],
+    "overrides": [],
+    "pull_accounting": true,
+    "form_order": [
+	"trusted_issuers_endpoint", "provider_did", "vc_type", "role_names"
```
Comment (ghost) [inline vc-trusted-issuer-data-service/package.json]: missing comma

File: vc-trusted-issuer-data-service/vc_til_service.py [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+            issuer_response.raise_for_status()
+        except HTTPError as e:
+            print('HTTP  ERROR')
+            print(e.request.body)
+            print(e)
+            print(e.response.text)
+            raise PluginError('Error when sending request to /issuer endpoint for creating issuer')
+
+    def get_usage_specs(self):
+        return UNITS
```
Comment (ghost) [inline vc-trusted-issuer-data-service/vc_til_service.py:303]: it looks like variable UNITS is not defined

File: vc-trusted-issuer-data-service/vc_til_service.py [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+from datetime import datetime, timedelta
+
+from urllib.parse import urlparse
+from urllib.parse import parse_qs
+
+from base64 import urlsafe_b64encode
+
+from django.conf import settings
+
+from wstore.asset_manager.resource_plugins.plugin import Plugin
```
Comment (ghost) [inline vc-trusted-issuer-data-service/vc_til_service.py:40]: how does the package 'wstore' being imported? My IDE can't find this dependency

File: vc-trusted-issuer-data-service/package.json (hunk 1 above)
Comment (dwendland) [reply vc-trusted-issuer-data-service/package.json]: fixed

File: vc-trusted-issuer-data-service/vc_til_service.py (hunk 2 above)
Comment (dwendland) [reply vc-trusted-issuer-data-service/vc_til_service.py:303]: fixed

File: vc-trusted-issuer-data-service/vc_til_service.py (hunk 3 above)
Comment (dwendland) [reply vc-trusted-issuer-data-service/vc_til_service.py:40]: The plugin is executed by the charging backend. There this package is defined.

**Review by ghost — APPROVED** [review]

**Review by wistefan — APPROVED** [review]

**Pull request merged**